File: gluecodium/src/test/resources/smoke/internal_fields/output/android/com/example/smoke/PublicFieldsMixedInit.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class PublicFieldsMixedInit {
    @NonNull
    public String publicField1;
    @NonNull
    public String publicField2;
    @NonNull
    String internalField;
    public PublicFieldsMixedInit(@NonNull final String publicField2) {
        this.publicField1 = "bar";
        this.publicField2 = publicField2;
        this.internalField = "foo";
    }
}

File: gluecodium/src/test/resources/smoke/internal_fields/output/android-kotlin/com/example/smoke/PublicFieldsMixedInit.kt
/*

 *
 */

@file:JvmName("PublicFieldsMixedInitExtensions")

package com.example.smoke


class PublicFieldsMixedInit {
    @JvmField var publicField1: String
    @JvmField var publicField2: String
    @JvmField internal var internalField: String



    constructor(publicField2: String) {
        this.publicField1 = "bar"
        this.publicField2 = publicField2
        this.internalField = "foo"
    }





}

